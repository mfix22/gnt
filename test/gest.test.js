/* global test, expect */
const graphicli = require('graphicli')
const schema = require('./_schema')

const gest = graphicli(schema)

test('Test types', async () => {
  const res = await gest(`
    {
      phone(value:"(507)-227-3854")
      date(value: "2017-05-07T20:38:02.854")
      card(value: "4111111111111111")
    }
  `)
  expect(res).toMatchSnapshot()
})
