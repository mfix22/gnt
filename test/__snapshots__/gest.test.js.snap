// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test types 1`] = `
Object {
  "data": Object {
    "card": Object {
      "cardType": "VISA",
      "isExpired": true,
      "number": "4111111111111111",
      "validCVV": false,
      "validExpiryMonth": false,
      "validExpiryYear": false,
    },
    "date": 1494189482,
    "email": null,
    "phone": "+15072273854",
    "state": "MN",
    "zip1": null,
    "zip2": "00631",
  },
  "errors": Array [
    [GraphQLError: Expected a value of type "NonEmptyString" but received: ],
    [GraphQLError: 00000 is not a valid zipcode],
  ],
}
`;
